script/files/boot/dts/overlays/poweroff-pins.dts

```diff
@@ -0,0 +1,26 @@
+/dts-v1/;
+
+/ {
+	compatible = "brcm,bcm2708";
+
+	fragment@0 {
+		target-path = "/";
+		__overlay__ {
+			poweroff_gpio: poweroff_gpio {
+				pinctrl-names = "default";
+				pinctrl-0 = <&poweroff_pins>;
+			};
+		};
+	};
+
+	fragment@1 {
+		target = <&gpio>;
+		__overlay__ {
+			poweroff_pins: poweroff_pins {
+				brcm,pins = <23 24>;
```

Comment:
Not sure if this should be 16 and 18 instead of 23 and 24. The syntax is a little ambiguous to me.
Technically, the pin # is 23 and 24, but the GPIO label are 16 and 18. I think 23 and 24 are just referred to in reference to the hardware location, but I could be wrong.
In LakeMaps/boat#16 I referenced a method by which the pins are configured as pull-up inputs that use pins 16 and 18.